translib/transformer/xfmr_intf.go

```diff
@@ -197,8 +210,18 @@ func performIfNameKeyXfmrOp(inParams *XfmrParams, requestUriPath *string, ifName
 	var err error
 	switch inParams.oper {
 	case DELETE:
+		if *requestUriPath == "/openconfig-interfaces:interfaces/interface/subinterfaces/subinterface" && subintfid != 0 {
+			return nil
```

Comment:
This case should return error instead of nil, may "subinterfaces not supported".
Also, this check is incomplete. We should do prefix match instead of exact match.
What if the requestUriPath is more specific than "../subinterfaces/subinterface" and subintfid != 0 ?
Maybe we should use strings.HasPrefix()

Reply:
Changed to check for prefix and returns an error now. Thank you for catching this!